cmd/annotations-checker/validate-annotations.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/google/go-github/v32/github" // GitHub API client library
+	// "github.com/ministryofjustice/cloud-platform-environments/pkg/namespace"
+	"golang.org/x/oauth2" // API requests authentication
+	// "google.golang.org/genproto/googleapis/api/annotations"
+	"gopkg.in/yaml.v2"
+)
+
+// Annotations structs to define the structure of annotations in the YAML file
```

Comment:
Let's get rid of all these comments, if you find a need for a comment try renaming the thing that needs a comment